tox.ini

```diff
@@ -54,6 +54,9 @@ setenv =
     dev: H_AUTHORITY = {env:H_AUTHORITY:lms.hypothes.is}
     dev: H_API_URL_PUBLIC = {env:H_API_URL_PUBLIC:http://localhost:5000/api/}
     dev: H_API_URL_PRIVATE = {env:H_API_URL_PRIVATE:http://localhost:5000/api/}
+    dev: MAILCHIMP_DIGESTS_SUBACCOUNT = {env:MAILCHIMP_DIGESTS_SUBACCOUNT:devdata}
```

Comment:
Is `MAILCHIMP_API_KEY` part of devdata?

Reply:
Not yet but I think it can be.

I don't think we can risk adding a real Mailchimp API key to devdata because the API keys can be used to send unlimited emails to any addresses using our Mailchimp account (unlimited until Mailchimp throttles or blocks us that is).

However you can create test API keys that let you really call the Mailchimp API but not actually send any emails, which I think would be a better local test than skipping the API call as it does currently.